src/components/first-section/first-section.styles.ts

```diff
@@ -0,0 +1,76 @@
+import { styled } from '@/lib/stitches'
+import handsTogether from '../../../public/hands-together.svg'
+
+export const SectionContainer = styled('section', {
+  display: 'flex',
+  height: '$160',
+  width: '100vw',
+  backgroundImage: `url(${handsTogether.src})`,
+  backgroundRepeat: 'no-repeat',
+  backgroundSize: 'cover',
+  backgroundPosition: 'center',
+  color: '$neutral-0',
+  alignItems: 'center',
+  justifyContent: 'center',
+  fontFamily: '$default',
+})
+
+export const ContentContainer = styled('div', {
+  display: 'flex',
+  width: '$176',
```

Comment:
Acho que está havendo uma confusão de espaçamento com width e height, são diferentes, espaçamentos são para usar nas distância entre elementos, como margins, paddings, width são variáveis mesmo, se for cadastrar todos de um site, vai precisar de um arquivo de spaces gigantesco